components/SwapTokenModal.vue

```diff
@@ -1,50 +1,76 @@
-<!-- Swap Tokens Modal -->
 <template>
-    <div class="modal fade" id="swapTokenModal" ref="swapTokenModal" tabindex="-1">
-        <div class="modal-dialog modalOfSwap" role="document" >
-        <div class="modal-content">
-            <div class="modal-header">
-            <h5 class="modal-title">{{ $t('swap_tokens') }}</h5>
-            <button type="button" class="close" data-dismiss="modal" aria-label="Close">
-                <span aria-hidden="true">&times;</span>
-            </button>
-            </div>
-            <div class="modal-body">
-            <iframe
-                id="swapWidgetkeychain"
-                title="Swap Tokens with Keychain"
-                :src= "swapWidgetUrl"
-                allow="clipboard-write"
-                width="400"
-                height="400"
-                style="border: none;"
-            />
-            </div>
+  <div class="modal fade" id="swapTokenModal" ref="swapTokenModal" tabindex="-1">
+    <div class="modal-dialog modalOfSwap" role="document">
+      <div class="modal-content">
+        <div class="modal-header">
+          <h5 class="modal-title">{{ $t('swap_tokens') }}</h5>
+          <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+            <span aria-hidden="true">&times;</span>
+          </button>
         </div>
+        <div class="modal-body">
+          <div class="loading-container" v-if="loading">
+            <div class="text-center">
+              <i class="fas fa-spinner fa-spin text-brand"></i>
+            </div>
+          </div>
+          <iframe
+            v-show="!loading"
```

Comment:
is there a reason you're using v-show instead of v-else? i suppose this case is the else of the above (loader/swap)?